martin-mbtiles/src/tile_copier.rs

```diff
@@ -222,25 +261,41 @@ impl TileCopier {
             self.run_query_with_options(
                 conn,
                 // Allows for adding clauses to query using "AND"
-                "INSERT INTO tiles SELECT * FROM sourceDb.tiles WHERE TRUE",
+                &format!(
+                    "INSERT {} INTO tiles SELECT * FROM sourceDb.tiles WHERE TRUE",
+                    match &self.options.on_duplicate {
+                        CopyDuplicateMode::Override => "OR REPLACE",
+                        CopyDuplicateMode::Ignore => "OR IGNORE",
+                        _ => "",
```

Comment:
don't use `_` when not required - better be explicit for all available options. This way when a new value is added to enum, it will stop compiling and you will know something needs to be handled